usaspending_api/download/management/commands/populate_monthly_delta_files.py

```diff
@@ -23,138 +23,167 @@
 from usaspending_api.references.models import ToptierAgency, SubtierAgency
 
 
-logger = logging.getLogger('console')
+logger = logging.getLogger("console")
 
 AWARD_MAPPINGS = {
-    'Contracts': {
-        'agency_field': 'agency_id',
-        'award_types': ['contracts', 'idvs'],
-        'column_headers': {
-            0: 'agency_id', 1: 'parent_award_agency_id', 2: 'award_id_piid', 3: 'modification_number',
-            4: 'parent_award_id', 5: 'transaction_number'
+    "Contracts": {
+        "agency_field": "agency_id",
+        "award_types": ["contracts", "idvs"],
+        "column_headers": {
+            0: "agency_id",
+            1: "parent_award_agency_id",
+            2: "award_id_piid",
+            3: "modification_number",
+            4: "parent_award_id",
+            5: "transaction_number",
+            6: "contract_transaction_unique_key",
```

Comment:
Were `contract_transaction_unique_key` and `assistance_transaction_unique_key ` requested? I read through the tickets so I understand the reasoning.
Did the deltas not originally contain `detached_award_proc_unique` and `afa_generated_unique`?

Reply:
They were not specifically requested for this ticket, but these files share a structure with normal downloads and thus were overlooked as part of https://federal-spending-transparency.atlassian.net/browse/DEV-2325 which actually contributed to the column misalignment issue.